### DISCUSSION TITLE
Unable to reorder an array field

### DISCUSSION THREAD

**Discussion by evanprovan** [sanity-io/sanity#3378] [Sanity Studio Support] [answered]
Using the dev-preview version of Sanity v3, I'm unable to reorder the items in an array field.

![image](https://user-images.githubusercontent.com/47653120/175837985-60757eec-9729-43b0-a811-b79d7bacc88b.png)

I have the following schema:

    defineField({
      name: 'materials',
      title: 'Materials',
      type: 'array',
      of: [
        {
          type: 'reference',
          to: [{type: 'materials'}],
        },
      ],
      options: {
        sortable: true,
      },
    }),

**Comment by p10e** [comment]
Hi @evanprovan, thanks for reporting! I was able to reproduce the issue using the schema definitions you shared. It seems to only happen when using an array of references. Using an array of objects instead correctly shows the drag handlers for me:
```
defineField({
  name: 'materials',
  title: 'Materials',
  type: 'array',
  of: [
    {
      to: [{type: 'someObject'}],
    },
  ],
  options: {
    sortable: true,
  },
}),
```

Will share this as a bug report with the team.

**Reply by ghost** [reply]
I added a similar bug report here: #3371

**Reply by fabien** [reply]
I’m having the same issue with arrays of references. On a side-note, the UI of those fields looks a bit off too (white-space, borders).

**Reply by davydog187** [reply]
@p10e is there any update on this bug?

**Reply by bryandowning** [reply]
This has been fixed for me as of `3.0.0-rc.0`

**Comment by bjoerge** [comment, marked as answer]
Fixed in latest RC. Thanks for reporting!
